app/assets/javascripts/controllers/cloud_tenant/cloud_tenant_form_controller.js

```diff
@@ -0,0 +1,51 @@
+ManageIQ.angular.app.controller('cloudTenantFormController', ['$http', '$scope', 'cloudTenantFormId', 'miqService', function($http, $scope, cloudTenantFormId, miqService) {
+  $scope.cloudTenantModel = { name: '' };
+  $scope.formId = cloudTenantFormId;
+  $scope.afterGet = false;
+  $scope.modelCopy = angular.copy( $scope.cloudTenantModel );
+  $scope.model = "cloudTenantModel";
+
+  ManageIQ.angular.scope = $scope;
+
+  if (cloudTenantFormId == 'new') {
+    $scope.cloudTenantModel.name = "";
+  } else {
+    miqService.sparkleOn();
+
+    $http.get('/cloud_tenant/cloud_tenant_form_fields/' + cloudTenantFormId).success(function(data) {
+      $scope.afterGet = true;
+      $scope.cloudTenantModel.name = data.name;
+
+      $scope.modelCopy = angular.copy( $scope.cloudTenantModel );
+      miqService.sparkleOff();
+    });
+  }
+
+  $scope.addClicked = function() {
+    miqService.sparkleOn();
+    var url = 'create/new' + '?button=add';
+    miqService.miqAjaxButton(url, true);
```

Comment:
Please change to `miqService.miqAjaxButton(url, $scope.cloudTenantModel)`  (I'm removing `miqAjaxButton` with `true` in https://github.com/ManageIQ/manageiq/pull/11324)